projects/plugins/jetpack/extensions/blocks/videopress/editor.js

```diff
@@ -292,6 +292,13 @@ const addVideoPressSupport = ( settings, name ) => {
 				},
 				default: [],
 			},
+			tracks: {
```

Comment:
we should map this attribute with [videoPressTracks](https://github.com/Automattic/jetpack/blob/99937fb86a9287312c5df88dcf9eb4f64184fdc4/projects/plugins/jetpack/extensions/blocks/videopress/editor.js#LL288C4-L288C20) instead of adding a new one.

Reply:
The thing is, they are not related to VideoPress at all. They are listed as `tracks` originally, before the extension from `core/video`, and since `tracks` is not in the attributes list, the attribute is removed and we end with a `<track ...` tag without the attribute.

![2023-05-15_16-50-50](https://github.com/Automattic/jetpack/assets/8486249/5548c69a-4c33-4c3f-86f4-0e393a95ca45)

Reply:
```
		const attributesDefinition = {
			...settings.attributes,
```
also works, but there may be unintended side effects doing this?

Reply:
The issue here is that the block is registered in general, irrespective of a particular block's data, but overwriting the attributes and removing some of the original ones. When a block does not use a VideoPress video, but a local one, this causes the rendering problem.

I updated the PR to add the original attributes before the new ones, so if an attribute is not used by v5 they are simply ignored, but normal videos do not break.